core/client/db/src/lib.rs

```diff
@@ -791,7 +791,7 @@ impl<Block> client::backend::Backend<Block, Blake2Hasher> for Backend<Block> whe
 			match self.storage.state_db.revert_one() {
 				Some(commit) => {
 					apply_state_commit(&mut transaction, commit);
-					let removed = best.clone();
+					let _removed = best.clone();
```

Comment:
Maybe we just want to remove it completely, when it is not used anyway.

Reply:
just wanted to silence the wanting - would keep full removal for a followup